com.unity.shadergraph/Editor/Drawing/Inspector/PropertyDrawers/PositionNodePropertyDrawer.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using UnityEditor.UIElements;
+using UnityEditor.Graphing;
+using UnityEditor.Graphing.Util;
+using UnityEngine.UIElements;
+
+namespace UnityEditor.ShaderGraph.Drawing.Inspector.PropertyDrawers
+{
+    [SGPropertyDrawer(typeof(PositionNode))]
+    class PositionNodePropertyDrawer : AbstractMaterialNodePropertyDrawer
+    {
+        internal override void AddCustomNodeProperties(VisualElement parentElement, AbstractMaterialNode nodeBase, Action setNodesAsDirtyCallback, Action updateNodeViewsCallback)
+        {
+            var node = nodeBase as PositionNode;
+            var previewField = new EnumField(node.m_TessellationOption);
+            var propertyRow = new PropertyRow(new Label("Tessellation Option"));
```

Comment:
Change the name here to "Source" instead.

Reply:
Done, thank you.